Food Receipe/Pages/index.html

```diff
@@ -1,96 +1,54 @@
-              alt="plate of delicious food"
-              class="result__image"
-            />
-            <h3 class="result__name">Meal Name</h3>
-            <button type="button" class="result__get-recipe">Get Recipe</button>
-          </div>
-          <div class="results__result">
-            <img
-              src="https://picsum.photos/300/200"
-              alt="plate of delicious food"
-              class="result__image"
-            />
-            <h3 class="result__name">Meal Name</h3>
-            <button type="button" class="result__get-recipe">Get Recipe</button>
-          </div>
-          <div class="results__result">
-            <img
-              src="https://picsum.photos/300/200"
-              alt="plate of delicious food"
-              class="result__image"
-            />
-            <h3 class="result__name">Meal Name</h3>
-            <button type="button" class="result__get-recipe">Get Recipe</button>
-          </div>
-        </div>
-        <dialog class="modal">
-          <p class="modal__close">&#10006;</p>
-          <h3 class="modal__name">Meal Name</h3>
-          <img
-            src="https://picsum.photos/300/200"
-            alt=""
-            class="modal__image"
-          />
-          <ul class="modal__category">
-            <li>category</li>
-            <li>category</li>
-          </ul>
-          <h4>Ingredients</h4>
-          <p class="modal__ingredients">Insert ingredients here.</p>
-          <h4>Instructions</h4>
-          <p class="modal__instructions">Insert instructions here.</p>
-          <a href="https://notion.so">Watch Video</a>
-        </dialog>
-        <footer>
-          <p>
-            Developed by
-            <a href="https://github.com/chingu-voyages/v46-tier1-team-01" target="_blank">Chingu Voyage46 Tier1 Team1</a> &copy;
-            2023
-          </p>
-        </footer>
-      </section>
-    </main>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="../styles.css" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Inter&display=swap">
+  <script src="https://kit.fontawesome.com/bb8e411a96.js" crossorigin="anonymous"></script>
+  <script src="../index.js" defer></script>
+  <title>Recipe Finder</title>
+</head>
+
+<body>
+  <header>
+    <h1>Recipe Hub</h1>
+    <p>Search recipes by ingredients</p>
+  </header>
+  <main>
+    <section class="search-section">
+      <form action="" method="get">
+        <input class="search__input" type="text" name="search" placeholder="Enter an ingredient" />
+        <button class="search-button" type="submit">
+          <i class="fa-solid fa-magnifying-glass"></i>
+          Search
+        </button>
+      </form>
+    </section>
+    <section class="results-section">
+      <h2>
+        Your search results for <span class="ingredient-searched"></span>
+      </h2>
```

Comment:
I suggest we add a class name to this h2 element, so we can use CSS and DOM manipulation to hide the line "Your search results for [ingredient]" by default, and only show this line after the user click the search button.

Reply:
Noted .  i will do in next part

Reply:
either that or we just select `.results-section h2 {}`

Reply:
@Shree-77  I saw you used all the classes I had created, that's great! 
I think it's useful to leave some mock-up (not pulled from the API) fake search results for styling purposes. In case we want to make changes to the way results are displayed we need to have mock-up results to see what our changes would look like. At least comment them out, but don't delete them. We can still use them until Sprint 6.

Reply:
also I saw that you deleted the dialog element, now we don't have the option to style the modal further (see previous comment) and the functionality is also missing.

Reply:
> either that or we just select `.results-section h2 {}`
We need a class name for the h2 element not just for CSS, but also for DOM manipulation in JavaScript. Unless there is a way for JavaScript to selecting the element with the parent class name that I am not aware of, we should stick to giving the h2 element its own class name.